### PR TITLE
Add changelog for SE-0128

### PR DESCRIPTION
**Pull request by trentxintong**
Add SE-0128 change log for 3.0 branch.

### CODE REVIEW

**Comment by trentxintong** [comment]
@swift-ci Please smoke test

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,33 @@ Note: This is in reverse chronological order, so newer entries are added to the
 Swift 3.0
 ---------
 
+* [SE-0128](https://github.com/apple/swift-evolution/blob/master/proposals/0128-unicodescalar-failable-initializer.md)
+
+  Some UnicodeScalar initializers (ones that are non-failable) now returns an Optional,
```
Comment (xwu) [inline CHANGELOG.md]: Nit: should be "now return an Optional"

**Comment by trentxintong** [comment]
@swift-ci Please smoke test

File: CHANGELOG.md (hunk 1 above)
Comment (trentxintong) [inline CHANGELOG.md]: :)

**Comment by trentxintong** [comment]
@swift-ci Please smoke test

**Comment by trentxintong** [comment]
@swift-ci Please smoke test

**Comment by trentxintong** [comment]
@swift-ci Please test

**Comment by trentxintong** [comment]
@swift-ci Please test OS X platform

**Pull request merged**